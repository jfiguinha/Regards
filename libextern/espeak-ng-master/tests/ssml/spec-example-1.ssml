<?xml version="1.0"?>
<!-- Example from SSML 1.1 specification, Appendix E
     https://www.w3.org/TR/speech-synthesis11/#AppE
-->
<speak version="1.1"
       xmlns="http://www.w3.org/2001/10/synthesis"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xsi:schemaLocation="http://www.w3.org/2001/10/synthesis
                   http://www.w3.org/TR/speech-synthesis11/synthesis.xsd"
       xml:lang="en-US">
  <p>
    <s>You have 4 new messages.</s>
    <s>The first is from Stephanie Williams and arrived at <break/> 3:45pm.</s>
    <s>
      The subject is <prosody rate="-20%">ski trip</prosody>
    </s>
  </p>
</speak>
