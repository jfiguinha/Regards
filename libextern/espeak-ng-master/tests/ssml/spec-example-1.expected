


juː hæv fˈoː͡ɹ nˈuː mˈɛsɪd͡ʒᵻz

ðə fˈɜːst ɪz fɹʌm stˈɛfəni wˈɪljəmz ænd ɚɹˈa͡ɪvd æt
θɹˈiː fˈoː͡ɹɾi fˈa͡ɪv pˌiːˈɛm

ðə sˈʌbd͡ʒɛkt ɪz skˈiː tɹˈɪp



